kernel/common/enumerable.rb

```diff
@@ -212,6 +212,12 @@ def slice_when(&block)
     end
   end
 
+  def chunk_while(&block)
+    raise ArgumentError, "wrong number of arguments (0 for 1)" unless block_given?
```

Comment:
Or how about "no block given" for consistency with `chunk` in https://github.com/rubinius/rubinius/blob/master/kernel/common/enumerable.rb#L8